src/bpftrace.cpp

```diff
@@ -185,6 +185,20 @@ int BPFtrace::add_probe(const ast::AttachPoint &ap,
 
           locations_from_dwarf = true;
         }
+
+        // Check if the function is inlined and warn
+        // the user about missing calls to inlined instances.
+        if (!config_.get(ConfigKeyBool::probe_inline)) {
+          auto inst_count =
+              dwarf->get_function_locations(probe.attach_point, true).size();
+          if (inst_count > 1) {
+            LOG(WARNING) << "Function \"" << probe.attach_point
```

Comment:
Is it easy to get numbers on the number of call sites and use the number of inline locations to indicate a percentage of inlined-ness?

Reply:
You'd like to compare how many time the function is inlined vs how many time it is called? What info does that give us?
I'm not sure about getting the number of call sites using lldb, I'll have a look...

Reply:
> You'd like to compare how many time the function is inlined vs how many time it is called? What info does that give us?

No, I'm just interested in call sites so I can understand, if possible, on how many call sites was this function inlined but it's not a blocker.